dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/enrollment/HibernateEnrollmentStore.java

```diff
@@ -279,7 +279,7 @@ private static String orderBy(List<Order> orders) {
       orderJoiner.add(
           order.getField() + " " + (order.getDirection().isAscending() ? "asc" : "desc"));
     }
-    return " order by " + orderJoiner;
+    return " order by " + orderJoiner + ", " + DEFAULT_ORDER;
```

Comment:
I guess there is no consequence if the same DEFAULT_ORDER is also part of the orderJoiner?

Reply:
Default order, aka entity id, it is not part of orderable fields, so it cannot ever be part of the orderJoiner here